webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -0,0 +1,200 @@
+import React, { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Skeleton, Label } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { TableText, TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import {
+  LongArrowAltUpIcon,
+  CheckIcon,
+} from '@patternfly/react-icons';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { MODULE_STREAMS_KEY } from './ModuleStreamsConstants';
+import { selectModuleStreamStatus } from './ModuleStreamsSelectors';
+import { useBulkSelect } from '../../../../Table/TableHooks';
+import { getHostModuleStreams } from './ModuleStreamsActions';
+import InactiveText from '../../../../../scenes/ContentViews/components/InactiveText';
+import TableWrapper from '../../../../../components/Table/TableWrapper';
+
+export const ModuleStreamsTab = () => {
+  const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
+  const {
+    id: hostId,
+    name: hostName,
+  } = hostDetails;
```

Comment:
you don't use hostDetails anywhere else, so you can do 
`
 const {
    id: hostId,
    name: hostName,
  }  = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
`

Reply:
also, now that I look at this, why are you not using the predefined selector in "HostDetailsSelectors.js"? 
export const selectHostDetails = state =>
  selectAPIResponse(state, HOST_DETAILS_KEY) || {};